release-please-config.json

```diff
@@ -0,0 +1,23 @@
+{
+    "bootstrap-sha": "1354831baeafb52965a8b2f5f80df5c43e414c99",
+    "exclude-paths": [
+      ".git",
+      ".idea",
+      ".github",
+      ".vscode"
+    ],
+    "release-type": "dotnet",
+    "include-component-in-tag": false,
```

Comment:
we probably need to tweak the following settings to keep the pre-release behaviour.
```
  // setting the type of prerelease in case of prerelease strategy
  "prerelease-type": "beta",

// when `manifest-release` creates GitHub Releases per package, create
  // those as "Prerelease" releases that have pre-major or prerelease versions.
  // absence defaults to false and all versions are fully Published.
  "prerelease": true
```

see [the reference](https://github.com/googleapis/release-please/blob/main/docs/manifest-releaser.md)

Reply:
Noticed the "prerelease-tye":"beta" is not supported, set the prerelease to true